mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -96,11 +96,7 @@ impl Bridge<WakingMemoryStore> {
                     &system_address,
                     settings.keep_alive(),
                     settings.clean_session(),
-                    Credentials::Anonymous(format!(
-                        "{}/$edgeHub/$bridge/{}",
-                        device_id,
-                        settings.name()
-                    )),
+                    Credentials::Anonymous(format!("{}/{}/$bridge", settings.name(), device_id,)),
```

Comment:
Shoot! I messed up the order... I think it is better if {device_id}/{$upstream}/bridge

Reply:
Fixing and then cherry picking to this PR